external/java/src/main/java/ch/epfl/dedis/lib/crypto/Bn256G2Point.java

```diff
@@ -1,49 +1,78 @@
 package ch.epfl.dedis.lib.crypto;
 
 import ch.epfl.dedis.lib.Hex;
+import ch.epfl.dedis.lib.exception.CothorityCryptoException;
 import com.google.protobuf.ByteString;
+import ch.epfl.dedis.lib.crypto.bn256.BN;
 
+import java.math.BigInteger;
 import java.util.Arrays;
 
 /**
  * The point used as a public key for a Bn256 signature
  */
 public class Bn256G2Point implements Point {
+    // TODO consider changing this to "bn256.g2"
```

Comment:
@Gilthoniel now that we have G1 points too, the marshalID for that is "bn256.g1". Do you think we should change the marshal IDs from "bn256.pt" to "bn256.g2" to be consistent with G1? this will need changes in kyber too

Reply:
Yes this is much better